pandas/tests/groupby/test_allowlist.py

```diff
@@ -86,19 +84,14 @@ def test_regression_allowlist_methods(raw_frame, op, level, axis, skipna, sort):
         frame = raw_frame.T
 
     if op in AGG_FUNCTIONS_WITH_SKIPNA:
-        grouped = frame.groupby(level=level, axis=axis, sort=sort)
+        grouped = frame.groupby("first", axis=axis, sort=sort)
         result = getattr(grouped, op)(skipna=skipna)
-        expected = getattr(frame, op)(level=level, axis=axis, skipna=skipna)
-        if sort:
-            expected = expected.sort_index(axis=axis, level=level)
-        tm.assert_frame_equal(result, expected)
     else:
-        grouped = frame.groupby(level=level, axis=axis, sort=sort)
+        grouped = frame.groupby("first", axis=axis, sort=sort)
         result = getattr(grouped, op)()
-        expected = getattr(frame, op)(level=level, axis=axis)
-        if sort:
-            expected = expected.sort_index(axis=axis, level=level)
-        tm.assert_frame_equal(result, expected)
+    # Previously compared to frame.op(level=...), but level removed in 2.0
+    # TODO: Assert something better
+    assert isinstance(result, DataFrame)
```

Comment:
I'd be good with a tracking issue for this. I'd guess this test is largely redundant, but we might not be testing the combination of all these arguments at the same time anywhere else.

Reply:
Agreed. I'll create an issue and add the reference here.